handler-host.go

```diff
@@ -172,6 +179,21 @@ VALUES (?, ?, ?, ?)
 	return nil
 }
 
+func (h *HostHandler) initStopChannels() {
+	h.die = make(chan bool, 2)
+	h.stopHeartbeat = make(chan bool, 2)
+	h.stopSend = make(chan bool, 2)
+	h.stopRead = make(chan bool, 2)
+}
+
+func (h *HostHandler) Close() {
```

Comment:
I would expect a function called Close to close the channels.

Reply:
Agreed